tests/test-sources/plugins/completion/cmp-all-sources.nix

```diff
@@ -21,7 +21,7 @@
 
             filterFunc = sourceName: !(elem sourceName disabledSources);
 
-            sourceNames = filter filterFunc (attrNames cmp-sources);
+            sourceNames = filter filterFunc (map (getAttr "name") cmp-sources);
```

Comment:
Would it be ok to just do:

```nix
additionalSources = [
  "otter"
  "some-other-source"
  "something-else"
  "not-in-sources.nix"
];

sourceNames = 
  filter filterFunc (map (getAttr "name") cmp-sources)
  ++ additionalSources;
```

This is similar to what we do in other tests where we don't have one definitive list to lookup.

Reply:
I can come up with a better solution if I tweak the test system to allow test-cases to be fully blown modules rather than only allowing plain-attrs configs.

Reply:
For me, it's fine to only test those extra sources in their respective test file. Because, they will most likely have one as they are "standalone" plugins.

Reply:
#1781 will allow me to do the same thing in the test as we do in `auto-enable.nix`, i.e. something like `attrNames config.cmpSourcePlugins`. No manual listing required.

Reply:
Okay, then why not !